telemetry/definitions/commonDefinitions.json

```diff
@@ -773,6 +773,61 @@
             "name": "codecatalyst_devEnvironmentWorkflowStep",
             "type": "string",
             "description": "Workflow step name"
+        },
+        {
+            "name": "profileNameStatus",
+            "type": "string",
+            "description": "Shows the status of Profile Name input validations under IAM Credentials",
```

Comment:
Does "Profile Name" refer to a specific UI element? We can make that more clear by starting the sentence with it:
```suggestion
            "description": "\"Profile Name\" input validation state under IAM Credentials",
```

Reply:
Yes it is an input field. For standardization I'll update this way of writing for all input fields.

Reply:
> For standardization I'll update this way of writing for all input fields

That will be a nice improvement! Should be a separate followup PR though.